src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4093,6 +4098,19 @@ export class StudyViewPageStore {
         default: []
     })
 
+    private withFusionDataChartCounts = remoteData<ClinicalDataCountSummary[]>({
+        invoke: async () => {
+            let molecularProfileSampleCounts = await internalClient.fetchMolecularProfileSampleCountsUsingPOST({
+                studyViewFilter: Object.assign({}, this.filters, { withFusionData: undefined })
+            });
+            return Promise.resolve(
```

Comment:
i'm pretty sure you don't need to Promise.resolve.  Just return getClinicalDataCountWithColorByCategoryCounts(...)

Reply:
got it. changed. thanks!